src/index.ts

```diff
@@ -445,7 +465,9 @@ export class Bigtable {
       {},
       baseOptions,
       {
-        servicePath: customEndpointBaseUrl || defaultBaseUrl,
+        servicePath:
+          customEndpointBaseUrl ||
+          `bigtable.${getDomain(options.BigtableClient)}`,
```

Comment:
If no universe domain and custom endpoint is provided then this will still be bigtable.googleapis.com.